python/scripts/remove_collaborator_locked_repos.py

```diff
@@ -121,7 +122,6 @@ def fetch_repositories(github_service: GithubService) -> list:
         collaborators_list, is_repository_locked = fetch_repository_data(
             repository_name, github_service
         )
-        time.sleep(1)
```

Comment:
Remove this to try and reduce the time the job takes to execute ⏰

If it fails due to API timeouts, and we find value in the job - more incentive to refactor into the GithubService ♻️